Makefile

```
@@ -101,6 +102,12 @@ docker-build-%:
 check-types:
 	cd src && pnpm run check-types
 
+run-integration-tests-dev:
+	cd integration-testing && pnpm run test:dev
+
+run-integration-tests-test:
+	cd integration-testing && pnpm run test:test
+
```

what do you think about using more generic task:

```suggestion
run-integration-tests-%:
	cd integration-testing && pnpm run test:$*
```